types/framework/entities/actor.d.ts

```diff
@@ -433,7 +436,7 @@ declare namespace Actor {
    * @typeParam D - Actor.Data to extract Item type from
    * @internal
    */
-  type OwnedItemData<D> = D extends Data<any, infer I> ? I : never;
+  type OwnedItemData<D extends Data> = D['items'][0];
```

Comment:
The difference between these is arbitrary. The main purpose of `Actor.OwnedItemData` is to ensure that the `Item.Data` used by the `Actor.Data` and `Item` supplied as type parameters to `Actor` are consistent.